dev/source/docs/debugging.rst

```diff
@@ -66,9 +66,18 @@ The stderr and stdout file handles are not available on ChibiOS, so fprintf is n
 hal.console->printf(...);
 .........................
 
+
 This output appears on the console.  For ChibiOS-based boards this is the first-defined SERIAL device.
 
 
+GPIO Assert Debugging
+..............
+
+It is possible to output debug assertions from ChibiOS (so the debug errors that you get if you build with --enable-asserts) using a GPIO pin. To do this you need to set "HAL_GPIO_PIN_FAULT" in hwdef.dat to the GPIO pin you want to use and then connect a logic analyzer to the pin. You will also need to configure your logic analyzer to interpret the serial stream and print readable information but most modern logic analyzers are able to do this automatically.
```

Comment:
you should just set FAULT pin in hwdef.dat, the HAL_GPIO_PIN_ is automatically added by the generator